### DISCUSSION TITLE
Discussion on Definition and Use of Preferred Name attribute in Agents

### DISCUSSION THREAD

**Discussion by ewommack** [ArctosDB/arctos#7960] [Agent Committee]
The @ArctosDB/diversity-and-inclusion discussed Agents in committee meeting this week, and believe that the definition and use of the [Preferred Name attribute ](https://arctos.database.museum/info/ctDocumentation.cfm?table=ctagent_attribute_type#preferred_name), should be looked at and possibly updated.

Some issues that may occur as Arctos continues to expand and diversify is that Preferred Name requires the presence of both a Given/First Name and a Last/Surname to create it. With the proposed changes to Name attributes (https://github.com/ArctosDB/arctos/issues/7477), there are going to be Agents that do not have a Given Name, and there may be ones that do not have a Surname. 

The committee also considered that idea that some people have different names that are the most used and functional, but that do not relate to their full complete name. Would it be better to have this has the Preferred Name instead, since it would be the name that is displayed most often?

Some possible suggestions that committee initially recorded were:
change to “display name”; difference between “full name” and “display name”

The committee will continue the discussion on this issue, and may bring it to the Code Table Committee or the UI Development Committee as the discussion develops.

**Comment by mkoo** [comment]
Thanks beth for the update. I'd like to move this to a Discussion. What would be useful would be for the committee to outline the functional need(s) to be addressed. I think there's some misunderstanding of some of the attribute functions which is understandable since your discussion was probably not focused on technicalities and I dont think Dusty was present.

So what are you trying to accomplish? to have Agents with a single name for Preferred Name? not just as an AKA? Not just an other_agent but as a person? Getting a clear functional needs and examples would be helpful then to have it brought to CT meeting for a technical assessment (next stage). I may not have that accurate btw so just ignore me if so!  thanks for meeting
